website2.0/static/css/custom.css

```diff
@@ -0,0 +1,23 @@
+/**
```

Comment:
After we migrated all needed files, we need to add ASF header here.

Reply:
yes Juzhiyuan